Storage/storage.h

```diff
@@ -78,11 +76,14 @@ namespace storage {
 #define CURRENT_STORAGE_STRUCT_TAG(base, alias)                                               \
   CURRENT_STRUCT(CURRENT_STORAGE_ADDER_##alias, base) {                                       \
     CURRENT_DEFAULT_CONSTRUCTOR(CURRENT_STORAGE_ADDER_##alias) {}                             \
-    CURRENT_CONSTRUCTOR( CURRENT_STORAGE_ADDER_##alias)(const base& value) : SUPER(value) {}  \
+    CURRENT_CONSTRUCTOR(CURRENT_STORAGE_ADDER_##alias)(const base& value) : SUPER(value) {}   \
   };                                                                                          \
-  CURRENT_STRUCT(CURRENT_STORAGE_DELETER_##alias, base) {                                     \
+  CURRENT_STRUCT(CURRENT_STORAGE_DELETER_##alias) {                                           \
+    CURRENT_FIELD(key, ::current::storage::sfinae::ENTRY_KEY_TYPE<base>); \
```

Comment:
For two reasons:
1) It's shorter
2) `map_.erase(sfinae::GetKey(object));` works out of the box :)